### PR TITLE
feat(kit): add helper `addAutoImport`

### PR DESCRIPTION
**Pull request by ricardogobbosouza**
### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

A helper for auto import (composables)

before
```js
nuxt.hook('autoImports:extend', (autoImports: AutoImport[]) => {
  autoImports.push({name: 'useExample', as: 'useExample', from: 'file-path'})
})
```

after
```js
import { addAutoImport } from '@nuxt/kit'

addAutoImport({name: 'useExample', as: 'useExample', from: 'file-path'})
```

### CODE REVIEW

**Comment by ricardogobbosouza** [comment]
wdyt @pi0 ?

File: packages/kit/src/composables.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import consola from 'consola'
+import { useNuxt } from './context'
+import { isNuxt2 } from './compatibility'
+import type { AutoImport } from '../../schema/src/types/imports'
+
+export function addComposables (composables: AutoImport | AutoImport[]) {
```
Comment (pi0) [inline packages/kit/src/composables.ts:6]: I would call this `addAutoImport` to be consistant with feature name

File: packages/kit/src/composables.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  const nuxt = useNuxt()
+
+  if (isNuxt2(nuxt)) {
+    consola.warn('Composables not available in Nuxt 2')
```
[the remark this answers is not in the document]
Comment (ricardogobbosouza) [reply packages/kit/src/composables.ts]: Ok, i used `assertNuxtCompatibility({ bridge: true })`

File: packages/kit/src/composables.ts (hunk 1 above)
Comment (ricardogobbosouza) [reply packages/kit/src/composables.ts]: Renamed, I left aliases for `addComposables`, `addComposablesDirs`

**Review by pi0 — APPROVED** [review]

**Pull request merged**